pvpanic/PVPanic Package/pvpanic_pack.props

```diff
@@ -0,0 +1,15 @@
+<!--
+***********************************************************************************************
+pvpanic_pack.props
+Enabling and customizing virtio build features
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="Platform">
+  <PropertyGroup>
+    <Feature_AlwaysDefaultVendor>false</Feature_AlwaysDefaultVendor>
+    <Feature_AdjustInf>true</Feature_AdjustInf>
+    <RHEL_COPYRIGHT_STARTING_YEAR>2015</RHEL_COPYRIGHT_STARTING_YEAR>
```

Comment:
looks good. I think that the best way is to get our patchset and apply your patch on top. This would be the simplest way.